library/core/src/fmt/mod.rs

```diff
@@ -971,9 +971,14 @@ pub trait UpperHex {
 /// `p` formatting.
 ///
 /// The `Pointer` trait should format its output as a memory location. This is commonly presented
-/// as hexadecimal.
+/// as hexadecimal. For more information on formatters, see [the module-level documentation][module].
 ///
-/// For more information on formatters, see [the module-level documentation][module].
+/// Printing of pointers is not a reliable way to discover how Rust programs are implemented,
```

Comment:
This is a complete sentence?
```suggestion
/// Printing of pointers is not a reliable way to discover how Rust programs are implemented.
```